### ISSUE TITLE
only authenticate medm thru auth button

### ISSUE THREAD

**Issue by stefco** [stefco/geco_vm#57]
see if possible to not open a terminal if working, and to pop open a gnome desktop window in the event of snafu.

**Comment by stefco** [comment]
- [x] need to add known hosts. use `ssh-keyscan` in the medm install script:

``` bash
ssh-keyscan lhoepics.ligo-wa.caltech.edu lloepics.ligo-la.caltech.edu >> .ssh/known_hosts
```

http://unix.stackexchange.com/questions/33271/how-to-avoid-ssh-asking-permission
- [x] also need to edit the `.desktop` files to have

```
Terminal=false
```

which will work and will prevent a terminal from popping up.

https://wiki.archlinux.org/index.php/Desktop_entries
- [x] finally, need to provide some sort of pop-up alert warning when something goes wrong (most likely unauthenticated)

use zenity:

http://ubuntuforums.org/showthread.php?t=876618

```
# tell the user that this thing is starting up
zenity --info --timeout=3 --text 'Starting (will take a few seconds)'
```

can even get username and password:

```
$ zenity --password --username
stc|blah
```

example here: 

https://help.gnome.org/users/zenity/stable/password.html.en

**Comment by stefco** [comment]
implemented (hopefully) in 11dec7e
